src/components/album/album-list/album-list-item/album-list-item.html

```diff
@@ -0,0 +1,17 @@
+<div class="album-list-item-wrapper">
+	<div class="album-list-item-img">
+		<img ng-src="{{$ctrl.album.images[0].url}}" alt="cover">	
+	</div>	
+	<div class="album-list-item-info">		
+		<div class="album-list-item-details">
+			<h3>{{ $ctrl.album.artists[0]["name"] }}</h3>
```

Comment:
Yeah I know that 'name' is a valid identifier (don't contain spaces or key words), so is not necessary using bracket notation